src/pages/Notes/Notes.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import NotesPage from "./Notes.page";
+import { GlobalContext } from "../../providers/Global/Global.provider";
+import { AuthContext } from "../../providers/Auth/Auth.provider";
+
+describe("<NotesPage />", () => {
+  test("should render all notes correctly", () => {
+    const contextValueGlobal = {
+      searchTerm: "",
+      activeNotes: [
+        {
+          color: "#0077ff",
+          id: "-Muh0xRHUL46QtpMkmce",
+          note: "notes teste",
+          status: true,
+        },
+      ],
+      getNotes: jest.fn(),
+    };
+    const contextValueAuth = {
+      userData: { user: { uid: "kjdhkd" } },
+    };
+    const { getByTitle } = render(
+      <AuthContext.Provider value={contextValueAuth}>
+        <GlobalContext.Provider value={contextValueGlobal}>
+          <NotesPage />
+        </GlobalContext.Provider>
+      </AuthContext.Provider>
+    );
+    const note = getByTitle("-Muh0xRHUL46QtpMkmce");
+    expect(note).toBeInTheDocument();
+  });
+
+  test("should no render note card correctly", () => {
+    const contextValueGlobal = {
+      activeNotes: [],
+      searchTerm: "",
+      getNotes: jest.fn(),
+    };
+    const contextValueAuth = {
+      userData: { user: { uid: "kjdhkd" } },
+    };
+    const { getByTitle } = render(
+      <AuthContext.Provider value={contextValueAuth}>
+        <GlobalContext.Provider value={contextValueGlobal}>
+          <NotesPage />
+        </GlobalContext.Provider>
+      </AuthContext.Provider>
+    );
+    const note = getByTitle("no-notes");
+    expect(note).toBeInTheDocument();
+  });
+});
```

Comment:
Here we can test to "create new note" by simulating the process to do so.
We can count how many notes we currently have, get the button to add a note and click it. Then fill the text input and trigger a click on the save button. Finally we can count again the number of notes rendered after saving. The expected result will be to have one element more than when we started.